### PR TITLE
Aligned and reformatted the text and details.

### PR DESCRIPTION
**Pull request by anselm94**
Also added links to license and acknowledgement

### CODE REVIEW

**Comment by forslund** [comment]
Hi, thanks for the pull request! Could you close and reopen pull request against the development branch?

And a couple of comments/questions:
- Headings shouldn't have a ':' at the end.
- Windows is missing from the list of platforms (an old issue but it might as well be fixed now!)
- Is the mimic -h really necessary in the readme?
- What is the reason you removed the chapter-lines under each heading?
- There is a missing "*/" in the if-statements section example. Can you fix this as well while you're at it?

**Comment by anselm94** [comment]
Hi,
I'll close and reopen the pull request into the 'development' branch. A little clarification needed. I don't understand what you mean by chapter-lines. :(

File: README.md [hunk 1]
```diff
@@ -1,258 +1,337 @@
-Mimic, the Mycroft TTS Engine
-==============================
+#Mimic - The Mycroft TTS Engine
```
Comment (forslund) [inline README.md]: These are the lines I talked about. The 
```==============================```

creates a horizontal line that the `#...` doesn't generate.

**Comment by anselm94** [comment]
Both `#...` and `##...` headers create an underline. Please take a look here https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet. Marking it as h1 makes it easy for search engines too :)

**Comment by forslund** [comment]
Ah, my bad. I blame my monitor, the contrast must have changed temporarily making the lines invisible =)

**Comment by anselm94** [comment]
:smile_cat:

**Pull request closed without merging**